testing/execution.ts

```diff
@@ -260,6 +262,30 @@ export async function executeFormulaOrSyncFromCLI({
         bundlePath,
       });
       printFull(chainedCommand ? chainedCommandResults : result);
+    } else if (formulaSpecification.type === FormulaType.Sync && chainedCommand?.type === FormulaType.Sync) {
```

Comment:
Ah hmm maybe we should move to a new enum for chained command types, it's not that intuitive that Sync chained with Sync means "incremental sync", at least to me.